backend/slack_commands/interactive/form_handler.py

```
@@ -292,5 +294,15 @@ def _wf_label(wf):
         return name
 
     @staticmethod
-    def _post_error(client, channel_id, text):
-        client.chat_postMessage(channel=channel_id, text=f":x: {text}")
+    def _resolve_reply_channel(client, channel_id: str, user_id: str) -> str:
+        if channel_id:
+            try:
+                client.conversations_info(channel=channel_id)
+                return channel_id
```

Ruff (0.16.0)

[warning] Consider moving this statement to an `else` block

(TRY300)